Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +494,107 @@ class HistoryPanel: SiteTableViewController, HomePanel {
+    }()
+
+    lazy var descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = DynamicFontHelper.defaultHelper.DeviceFontSmallHistoryPanel
+        label.textColor = TwoLineCellUX.DetailTextColor
+        return label
+    }()
+
+    lazy var imageView: UIImageView = {
+        let image = UIImage(named: "panelIconSyncedTabs")!
+        return UIImageView(image: image)
+    }()
+
+    lazy var chevron: ChevronView = {
+        let chevron = ChevronView(direction: .Right)
+        chevron.tintColor = HistoryPanelUX.SyncedTabsCellChevronColor
+        chevron.lineWidth = 3.0
+        return chevron
+    }()
+
+    lazy var topBorder: UIView = self.createBorderView()
+    lazy var bottomBorder: UIView = self.createBorderView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        userInteractionEnabled = true
+        backgroundColor = SiteTableViewControllerUX.HeaderBackgroundColor
+
+        addSubview(topBorder)
+        addSubview(bottomBorder)
+        addSubview(chevron)
+        addSubview(titleLabel)
+        addSubview(imageView)
+        addSubview(descriptionLabel)
+
+
+        imageView.snp_makeConstraints { make in
+            make.left.equalTo(self).offset(TwoLineCellUX.BorderViewMargin)
+            make.centerY.equalTo(self)
+            make.width.equalTo(HistoryPanelUX.SyncedTabsCellImageSize)
+        }
+
+        chevron.snp_makeConstraints { make in
+            make.right.equalTo(self).offset(-HistoryPanelUX.SyncedTabsCellChevronInset)
+            make.centerY.equalTo(self)
+            make.size.equalTo(HistoryPanelUX.SyncedTabsCellChevronSize)
+        }
+
+        topBorder.snp_makeConstraints { make in
+            make.left.right.equalTo(self)
+            make.top.equalTo(self).offset(-0.5)
+            make.height.equalTo(0.5)
+        }
+
+        bottomBorder.snp_makeConstraints { make in
+            make.left.right.bottom.equalTo(self)
+            make.height.equalTo(0.5)
+        }
+
+        descriptionLabel.snp_makeConstraints { make in
+            make.left.equalTo(imageView.snp_right).offset(TwoLineCellUX.BorderViewMargin)
+            make.centerY.equalTo(self).offset(10)
+        }
+
+        updateConstraints()
+    }
+
+    private func createBorderView() -> UIView {
+        let view = UIView()
+        view.backgroundColor = SiteTableViewControllerUX.HeaderBorderColor
+        return view
+    }
+
+    override func updateConstraints() {
+        super.updateConstraints()
+
+        titleLabel.snp_remakeConstraints { make in
+            make.left.equalTo(imageView.snp_right).offset(TwoLineCellUX.BorderViewMargin)
+            make.centerY.equalTo(self)
+        }
+
+        if !descriptionLabel.hidden {
+            titleLabel.snp_remakeConstraints { make in
+                make.left.equalTo(imageView.snp_right).offset(TwoLineCellUX.BorderViewMargin)
+                make.centerY.equalTo(self).offset(-10)
+            }
+        }
```

Comment:
Nit: Instead of remaking constraints twice on the same view, why not condense this?

```swift
 titleLabel.snp_remakeConstraints { make in
    make.left.equalTo(imageView.snp_right).offset(TwoLineCellUX.BorderViewMargin)
    make.centerY.equalTo(self).offset(descriptionLabel.hidden ? 0 : -10)
}
```